render_delegate/render_param.cpp

```diff
@@ -217,4 +245,44 @@ void HdArnoldRenderParam::WriteDebugScene() const
     AiParamValueMapDestroy(params);
 }
 
+double HdArnoldRenderParam::GetElapsedRenderTime() const
+{
+    const auto t0 = _renderStartTime.load();
+    const auto t1 = std::chrono::system_clock::now();
+    const auto delta = std::chrono::duration_cast<std::chrono::milliseconds>(t1 - t0);
+
+    return delta.count();
+}
+
+void HdArnoldRenderParam::StartRenderMsgLog()
+{
+    _msgLogCallback = AiMsgRegisterCallback(MsgStatusCallback, AI_LOG_STATUS, this);
+}
+
+void HdArnoldRenderParam::StopRenderMsgLog()
+{
+    AiMsgDeregisterCallback(_msgLogCallback);
```

Comment:
we might just want to set _msgLogCallback to nullptr here, just to be sure we don't try to deregister the same callback multiple times.

Reply:
Whoops, good spot